### ISSUE TITLE
No theme selection

### ISSUE THREAD

**Issue by rafkan20** [chaptergy/noctis-grey#8]
Hi, great theme.
I have been using nostis-grey for a long time.
Until v1.3.5, the icons were white for me.
![jazz](https://images89.fotosik.pl/659/7f3551cb6afacfff.jpg)
In the profile option I was able to select the "nostis-grey" theme.
![jazz](https://images89.fotosik.pl/659/0c58028603bd6c6c.jpg)
After upgrading to v1.3.6, despite selecting the theme nostis-grey, the theme does not change.
![jazz](https://images92.fotosik.pl/659/03c0a13a953fcc56.jpg)
When "dark" is checked, the theme does not change to nostis-gray (it only becomes dark).

![jazz](https://images90.fotosik.pl/657/0ec20a604680aca5.jpg)
![jazz](https://images92.fotosik.pl/659/82407bfc3298f234.jpg)
I am using HA version 2023.4.4 (I updated to 2023.4.5 today).

**Comment by z-master42** [comment]
The difference between 1.3.5 and 1.3.6 is that the icon and some text colors have been changed. My icons were white before and now they are in this grayish blue. I found the white icons also better, especially with activated entities. The contrast was simply nicer. At the moment I have to stay on 1.3.5.

I use a lot of [slider button cards](https://github.com/custom-cards/slider-button-card) and they look like this when switched on:
1.3.5
![1 3 5](https://user-images.githubusercontent.com/66380371/233162116-bfc62c3c-1545-4081-acae-c81c82b105a3.png)
1.3.6
![1 3 6](https://user-images.githubusercontent.com/66380371/233162124-9e62f7f5-2d14-43c8-9d69-76b19b49a99c.png)
